aries-site/src/examples/templates/popover/PopoverInlineExample.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useRef, useContext } from 'react';
+import { Popover } from 'aries-core';
+import {
+  Anchor,
+  Grid,
+  Card,
+  CardBody,
+  Button,
+  Box,
+  Paragraph,
+  Text,
+  Heading,
+  ResponsiveContext,
+} from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+const device = [
+  {
+    title: 'Require Assignments',
+    devices: '10',
+  },
+  {
+    title: 'Require Subscriptions',
+    devices: '10',
+  },
+  {
+    title: 'Assigned & Subscribed',
+    devices: '0',
+  },
+  {
+    title: 'Total Devices',
+    devices: '10',
+  },
+];
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto', 'auto'],
+};
+
+// should we disable ?
+// eslint-disable-next-line react/prop-types
+const CardDevices = ({ title, devices }) => (
+  <Card>
+    <CardBody>
+      <Text>{title}</Text>
+      <Text>{devices}</Text>
+    </CardBody>
+  </Card>
+);
+
+export const PopoverInlineExample = () => {
+  const breakpoint = useContext(ResponsiveContext);
+  const [showPopover, setShowPopover] = useState(false);
+  const targetRef = useRef();
+
+  const handleButtonClick = () => {
+    setShowPopover(prev => !prev);
+  };
+
+  const handleClosePopover = () => {
+    setShowPopover(false);
+  };
+
+  return (
+    <Box gap="medium">
+      <Box
+        justify="between"
+        direction={!['xsmall', 'small'].includes(breakpoint) ? 'row' : 'column'}
+      >
+        <Box>
+          <Heading margin="none" level={2}>
+            Inventory
+          </Heading>
+          <Box direction="row">
+            <Text size="large">
+              View all available devices or add new devices.
+            </Text>
+            <Button
+              align="center"
+              justify="start"
+              icon={<CircleInformation size="small" />}
+              onClick={handleButtonClick}
+              ref={targetRef}
+              a11yTitle="Informational help"
+              aria-expanded={showPopover}
+            />
+          </Box>
+        </Box>
+        {showPopover && targetRef.current && (
+          <Popover
```

Comment:
The close button aria-label on this example is really long

Reply:
@jcfilben - Do you have an alternate suggestion?

Reply:
Instead of reading out the paragraph can we give a quick tagline of what the popover is (ex popover with details about ___). Then if the screenreader user wants to know more they could have their screenreader read all the text in the popover